packages/core/src/actions/codegen/createWriteContract.ts

```diff
@@ -86,7 +86,7 @@ export type CreateWriteContractReturnType<
               | undefined
           }
         : Compute<ChainIdParameter<config, chainId>>) &
-      ConnectorParameter & { __mode?: 'prepared' }
```

Comment:
Might be good to leave `__mode` here and just add a `@deprecated` comment so people that upgrade don't experience unexpected type errors if they were passing this in directly (to skip simulation).